Athentiaction/Login/login.css

```diff
@@ -0,0 +1,54 @@
+.login {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 90vh;
+  }
+  
+  .login-container {
+    box-shadow: 0 0 5px var(--grey-color);
+    padding: 2rem 4rem;
+    border-radius: 10px;
+  }
+  
+  .login-btn {
+    background-color: var(--primary-color);
+    color: var(--white-color);
+    width: 100%;
+  }
+  
+  .login-btn:hover {
+    background-color: var(--primary-hover);
+  }
+  
+  .center {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
You can create this class in your component library and use it across all other projects.

Reply:
ok